cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -589,14 +650,23 @@ def test(
     submission_df.sort_values(by=["Id"]).to_csv(
         os.path.join(working_dir, hp.local_submission_csv), index=False
     )
+    # predictions are saved to a CSV file.
     print("Saved predictions to %s" % hp.local_submission_csv)
 
     return working_dir
 
-
+# %%
+# Defining the Spark Task
+# =====================
+#
+# Flyte provides an easy-to-use interface to specify Spark-related attributes.
+# Refer to `this <https://docs.flyte.org/projects/cookbook/en/latest/auto/case_studies/ml_training/spark_horovod/index.html#flyte-and-spark>`__ for the installation instructions.
```

Comment:
To set up Spark, refer to :ref:`flyte-and-spark`.

Can we have this as a note below the next line?

Reply:
Also, please add `.. _flyte-and-spark:` before the Flyte and Spark section in README.